docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: "3.8"
+
+services:
+  ingress-proxy:
+    container_name: dissolution-ingress-proxy
+    image: traefik:v2.2
+    command: --api.insecure=true --providers.docker
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock # So that Traefik can listen to the Docker events
+    networks:
+      - chs-dev
+    ports:
+      - 80:80
+      - 8080:8080 # Web UI (enabled by --api.insecure=true)
+  dissolution-web:
+    build: .
+    image: dissolution-web:1.0
+    labels:
+      - traefik.http.routers.lfp-appeals-frontend.rule=Host(`chs-dev`) && PathPrefix(`/dissolution-web/`)
+    ports:
+      - 3000:3000
+    networks:
+      - chs-dev
+    working_dir: /app
+    environment:
+      - CDN_HOST=cdn.chs-dev
+      - CACHE_SERVER=0.0.0.0:6379
+      - CACHE_DB=0
+      - CACHE_PASSWORD=''
+
+      - COOKIE_NAME=SID
+      - COOKIE_DOMAIN=chs-dev.internal
+      - COOKIE_SECURE_ONLY=0
+      - COOKIE_SECRET=ChGovUk-XQrbf3sLj2abFxIY2TlapsJ
+      - DEFAULT_SESSION_EXPIRATION=3600
+
+      - ACCOUNT_WEB_URL=http://web.chs-dev.internal:4000
+
+      # - FILE_TRANSFER_API_URL:
+      # - FILE_TRANSFER_API_KEY:
```

Comment:
Can we remove these comments? Or do we need to keep them?

Reply:
I agree, let's only keep the variables we need